### ISSUE TITLE
[v5] Page goes to infinite load when open in a new tab launch filtered by new temp filter

### ISSUE THREAD

**Issue by Shakevg** [reportportal/reportportal#1282]
**Describe the bug**
Page goes to infinite load when open in a new tab launch filtered by new temp filter. Looks like a problem that not existing filter number used in GET request

**To Reproduce**
1. Click on some launch attribute to filter
=> OK: Temp filter added on top
2. Open some filtered launch or tests on a new tab
=> FAIL: Infinite spinning, error return from the server due filter number -1 (see details below)

**Expected behavior**
Launch data is displayed on a new tab

**Screenshots**
![image](https://user-images.githubusercontent.com/42133199/105385609-97c85a00-5c24-11eb-8210-78848c47ac83.png)

**Desktop (please complete the following information):**
 - OS: [Windows]
 - Browser [e.g. chrome]
 - Version [87.0.4280.141]

**Additional context**
Information from Fiddler:
```
Request: 
GET https://rp/api/v1/project/filter/-1 HTTP/1.1
Response: 
HTTP/1.1 404
{"errorCode":40421,"message":"User filter with ID '-1' not found on project 'project'. Did you use correct User Filter ID?"}
```

**Comment by VolhaKarenko** [comment]
Issue is reproducible, added to backlog

**Comment by VolhaKarenko** [comment]
Fixed in version 5.4
